### ISSUE TITLE
How hard is it to port Hermes to Windows?

### ISSUE THREAD

**Issue by ufuksarp** [HermesApp/Hermes#314]
I tried all the Pandora clients for Windows, they all are ugly/cumbersome/lacking features.
How hard is it to port Hermes to Windows?

**Comment by nriley** [comment]
Agreed — I settled on a console-based client on Windows (pianobar) after having tried and hated all the GUI options.  I don't think there'd be much code in Hermes that would be easily portable directly; there are a few compatibility layers for Cocoa on Windows that might work, but they'd still require quite a lot of work I think.  This wouldn't cover the lower levels like audio and networking, which would probably have to be completely reimplemented.
